libs/api/domains/notifications/src/lib/notifications.model.ts

```diff
@@ -116,3 +109,21 @@ export class MarkNotificationReadResponse {
   @Field(() => Notification)
   data!: Notification
 }
+
+@ObjectType()
+export class MarkAllAsSeenResponse {
```

Comment:
Shouldn't this type be prepended by "Notifications"

So `NotificationsMarkAllAsSeenResponse`